TrackingTools/PatternTools/src/TrajectoryExtrapolatorToLine.cc

```diff
@@ -13,7 +13,8 @@ TrajectoryStateOnSurface TrajectoryExtrapolatorToLine::extrapolate(const FreeTra
   DeepCopyPointerByClone<Propagator> p(aPropagator.clone());
   p->setPropagationDirection(anyDirection);
 
-  FreeTrajectoryState fastFts(fts.parameters());
+//  FreeTrajectoryState fastFts(fts.parameters());
+  FreeTrajectoryState fastFts = fts;  
```

Comment:
-1
This is clearly a computational performance regression, need to be properly quantified.